ui/app/templates/components/alert-popup.hbs

```diff
@@ -11,9 +11,12 @@
         {{this.type.text}}
       </div>
       {{#if this.message}}
-        <p class="message-body {{if @isPreformatted "pre"}}" data-test-flash-message-body="true" test="another attribute">
-          {{this.message}}
-        </p>
+        {{! template-lint-disable block-indentation}}
+        <p
+          class="message-body {{if @isPreformatted "pre"}}"
+          data-test-flash-message-body="true"
+          test="another attribute"
```

Comment:
it looks like this `test` attribute may have been left over from the upgrade?

Reply:
perhaps, good catch! I'll remove